app/assets/javascripts/quick_create.js

```diff
@@ -0,0 +1,93 @@
+function addToAutocomplete(json) {
+  var source = JSON.parse($("#" + json.autocomplete_id).attr("data-autocomplete-source"));
+  source.unshift(json.name)
```

Comment:
yeah that comment could be ignored, I don't have a habit to lean on .erb templates, but as it is now, we shift the heftier bits onto .erbs / rails so makes sense.